src/Cake.NuGet.Tests/Unit/NuGetModuleTests.cs

```diff
@@ -65,6 +98,26 @@ public void Shouls_Register_The_NuGet_Package_Installer()
                 fixture.Builder.Received(1).As<IPackageInstaller>();
                 fixture.Builder.Received(1).Singleton();
             }
+
+#pragma warning disable xUnit1004 // Test methods should not be skipped
+            [Fact(Skip = "Don't know why this is failing...")]
+#pragma warning restore xUnit1004 // Test methods should not be skipped
+            public void Should_Register_The_In_Process_NuGet_Package_Installer_If_Set_In_Configuration()
+            {
+                // Given
+                var fixture = new NuGetModuleFixture<NuGetPackageInstaller>();
+                fixture.Configuration.SetValue(Constants.NuGet.UseInProcessClient, bool.TrueString);
+                var module = fixture.CreateModule();
+
+                // When
+                module.Register(fixture.Registrar);
+
+                // Then
+                fixture.Registrar.Received(1).RegisterType<Install.NuGetPackageInstaller>();
+                fixture.Builder.Received(1).As<INuGetPackageInstaller>();
```

Comment:
@cake-build/cake-team can anyone tell me why this test is failing here?

Reply:
see above comment